qa/tasks/ceph_fuse.py

```diff
@@ -86,6 +86,24 @@ def task(ctx, config):
             client.1:
               mount_subvol_num: 1
 
+    Example for client mount with custom client feature set
+
+        tasks:
+        - ceph:
+        - ceph-fuse:
+            client.0:
+              client_features: 0,1,2,3,4,19,20,21 # only these feature bits
+
+    Example for client mount with custom client feature set
+
+        tasks:
+        - ceph:
+        - ceph-fuse:
+            client.0:
+              client_feature_range: 21 # everything except CEPHFS_FEATURE_MDS_AUTH_CAPS_CHECK
```

Comment:
Looking back, I regret not making the config not simply subtract bits out of `CEPHFS_FEATURES_CLIENT_SUPPORTED`. The current design will require eventually updating this when other bits are added that are required (beyond the DUMMY bit).

Reply:
(It's not necessary to change this now. Perhaps a future cleanup.)

Reply:
That's an enhancement we should hand over to a junior dev.